src/ResultsHouseblock/ResultsHouseblock.js

```diff
@@ -5,40 +5,28 @@ import '../App.scss';
 import PropTypes from 'prop-types';
 import { withRouter } from 'react-router';
 import { getHoursMinutes, properCaseName } from "../stringUtils";
+import DatePickerInput from "../DatePickerInput";
+import moment from 'moment';
 
 class ResultsHouseblock extends Component {
-  buildTableForRender () {
-    const offenders = this.props.houseblockData && this.props.houseblockData.map((row, index) => {
-      const mainActivity = row.activity;
-      return (
-        <tr key={mainActivity.offenderNo} className="row-gutters">
-          <td className="row-gutters">{properCaseName(mainActivity.lastName)}, {properCaseName(mainActivity.firstName)}</td>
-          <td className="row-gutters">{mainActivity.cellLocation}</td>
-          <td className="row-gutters">{mainActivity.offenderNo}</td>
-          <td className="row-gutters">{mainActivity.eventDescription}</td>
-          <td className="row-gutters">{row.others &&
-            <ul>{row.others.map((e, index) => {
-              return <li key={mainActivity.offenderNo + '_' + index}>{e.eventDescription} - {getHoursMinutes(e.startTime)}</li>;
-            })}</ul>
-          }</td>
-          <td><input type="checkbox" /></td>
-          <td><input type="checkbox" /></td>
-          <td>TODO</td>
-          <td><input type="checkbox" /></td>
-          <td><input type="checkbox" /></td>
-        </tr>
-      );
-    });
-    return offenders;
+  isToday () {
+    const searchDate = moment(this.props.date, 'DD/MM/YYYY');
+    return searchDate.isSame(moment(), "day");
+  }
+
+  olderThan7Days () {
+    const searchDate = moment(this.props.date, 'DD/MM/YYYY');
+    const days = moment().diff(searchDate, "day");
+    return days > 7;
   }
 
   render () {
     const housingLocations = this.props.locations ? this.props.locations.map((loc, optionIndex) => {
-      return <option key={`housinglocation_option_${loc}`} value={loc}>{loc}</option>;
+      return <option key={`housinglocation_option_${loc}`}>{loc}</option>;
     }) : [];
 
     const locationSelect = (
-      <div className="pure-u-md-7-12">
+      <div className="pure-u-md-4-12" style={{ width: '30%', display: 'inline-block' }}>
```

Comment:
eek - maybe change to a class rather than inline styles?

Reply:
Ah yes, forgot I left that in ! Will push again

Reply:
i'll approve if you want to change it later